decidim-elections/app/views/decidim/elections/admin/questions/_form.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="questionnaire-questions">
+  <div class="row column text-center">
+    <button type="button" class="button button__sm button__secondary collapse-all"><%= t("collapse", scope: "decidim.forms.admin.questionnaires.form") %></button>
+    <button type="button" class="button button__sm button__secondary expand-all"><%= t("expand", scope: "decidim.forms.admin.questionnaires.form") %></button>
+  </div>
+
+  <%= fields_for "questions[#{blank_question.to_param}]", blank_question do |question_form| %>
+    <script type="text/template" class="decidim-question-template decidim-template" id="question-template">
+      <%= render "decidim/elections/admin/questions/question",
+                 form: question_form,
+                 id: tabs_id_for_question(blank_question),
+                 editable: true,
+                 response_option_template_selector: "#response-option-template-dummy" %>
+    </script>
+    <%= render "decidim/elections/admin/questions/response_option_template", form: question_form, editable: true, template_id: "response-option-template-dummy" %>
+  <% end %>
+
+  <div class="questionnaire-questions-list flex flex-col py-6 gap-6 last:pb-0">
+    <ul class="draggable-list js-connect js-list-questions">
+      <% @form.questions.each_with_index do |question, index| %>
+        <%= fields_for "questions[]", question do |question_form| %>
+          <li class="draggable-item js-question-block" draggable="true">
```

Comment:
There is a small problem with this draggable, as there is no gap-6 present : 

This is from current survey module. 
![image](https://github.com/user-attachments/assets/bdd9e9e4-f427-4d5a-ad43-c74cce82ad4a)

Reply:
The spacing is not resolved.